db/sqldb/portfolio.go

```diff
@@ -69,7 +72,9 @@ func (d *DB) PortfoliosByOwner(tx db.Tx, ownerID pacta.OwnerID) ([]*pacta.Portfo
 	if err != nil {
 		return nil, fmt.Errorf("translating rows to portfolios: %w", err)
 	}
-	return pvs, nil
+	// Note the map interface here is ~required in the deserialization process to track multiple membersihps,
+	// so we're not just converting to a map and back.
```

Comment:
I appreciate getting out in front of the inevitable review comment, though instead of having the SQL query repeatedly returns the whole portfolio alongside each group it is in, you could split it into two queries, one for the portfolios, one for the memberships. But that approach still has a mapping step, so your point stands, LGTM

Also niiiit on `s/membersihps/memberships`

Reply:
Yay! GC on the typo.